source/gui/settingsDialogs.py

```diff
@@ -2996,10 +2996,7 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		# Audio settings panel.
 		label = _("Volume of other applications")
 		self.appSoundVolSlider: nvdaControls.EnhancedInputSlider = sHelper.addLabeledControl(
-			label,
-			nvdaControls.EnhancedInputSlider,
-			minValue=0,
-			maxValue=100
```

Comment:
please restore this and add a trailing comma to `maxValue` like suggested earlier